deploy/template.yaml

```diff
@@ -616,7 +622,7 @@ Resources:
               Value: !If
                 - IsBuild
                 - "development"
-                - !If [IsDevelopment, !Sub "development", !Ref "AWS::NoValue"]
```

Comment:
please note this line failed in cfn-lint not sure why a !Sub was added to a string with no substitutions, can this be verified ?